control-plane/src/modules/workflows/workflow-messages.ts

```diff
@@ -244,7 +244,7 @@ export const getRunMessagesForDisplay = async ({
       displayableContext: workflowMessages.metadata,
     })
     .from(workflowMessages)
-    .orderBy(asc(workflowMessages.created_at))
+    .orderBy(desc(workflowMessages.created_at))
```

Comment:
This was returning the "first" 100 not the last 100.

https://github.com/inferablehq/inferable/issues/372